lib/scrabble.rb

```diff
@@ -1,7 +1,20 @@
+require 'pry'
 class Scrabble
 
   def score(word)
-    1
+    letters = word.upcase.split("")
+
+    total_score = 0
+    letters_score = []
+    letters.each do |letter|
+      letters_score << point_values.values_at(letter)
+    end
+    # binding.pry
+    letters_score.join.to_i
```

Comment:
Don't think we need to join these scores (assuming this is an array of integers). Just need to sum. Ruby 2.4 gives us `#sum` for arrays. In earlier versions we can use `#reduce(:+)`.